src/components/Modal/Translate.vue

```diff
@@ -94,6 +112,7 @@ export default {
 			canDetect: loadState('text', 'translation_can_detect'),
 			loading: false,
 			error: null,
+			disableFromLanguageSelect: true,
```

Comment:
What's the purpose of `disableFromLanguageSelect`? It seems to be set to `false` immediately by `onInputFocus` when the modal is opened. So maybe a leftover that can go away?

Reply:
It is for fixing the autofocus of first select box since we expect to autofocus on the first input textarea.

Reply:
@mejo- Is it all good now?